content/tr/index.md

```diff
@@ -2,21 +2,21 @@
 
 [ES6](es6.html)'ı Node topluluğuna kazandırdık
 
-[io.js](https://github.com/iojs/io.js), [node.js](https://nodejs.org/)'e dayalı  ve [npm](https://www.npmjs.org/)'le uyumlu bir platformdur&#8482;.
+[io.js](https://github.com/iojs/io.js), [node.js](https://nodejs.org/)&#8482;'e dayalı  ve [npm](https://www.npmjs.org/)'le uyumlu bir platformdur.
 
 [![io.js](../images/1.0.0.png)](https://iojs.org/dist/v1.4.1/)
 
-[Sürüm 1.4.1](https://iojs.org/dist/v1.4.1/)
+[Sürüm 1.3.0](https://iojs.org/dist/v1.4.1/)
```

Comment:
Please double check that the io.js and V8 version numbers match up to what's currently on the main site. In the VERY near future I hope to make this automated (via a variable) so you don't have to worry about details like this on future pull requests.